backend/config.py

```
@@ -0,0 +1,98 @@
+        try:
+            config.llm = LLMConfig()
+        except (ValidationError, ValueError):
+            logging.warning(
+                "LLM config not available. Chat endpoints will be disabled."
+            )
+
+        # R2設定のロード
+        try:
+            config.r2 = R2Config()
+        except (ValidationError, ValueError):
+            logging.error("R2 config is required for backend operation")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)